TimeTrackerApp/TimeTrackerApp/Data/DataAccessLayer.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DataAccessLayer.swift
+//  TimeTrackerApp
+//
+//  Created by Mine Rala on 29.09.2022.
+//
+
+import UIKit
+import CoreData
+
+final class DataAccessLayer {
+    static func getContext() -> NSManagedObjectContext {
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        return appDelegate.persistentContainer.viewContext
+    }
```

Comment:
Big O'su 1 olan = O(1) olan değerleri computed variable yapmak daha verimli çalışıyor.
https://medium.com/swift-india/functions-vs-computed-property-what-to-use-64bbe2df3916